### PR TITLE
Add JavBus.yml for https://www.javbus.com

### PR DESCRIPTION
**Pull request by benny502**

### CODE REVIEW

File: scrapers/JavBus.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          - replace:
+            - regex: ^(.*? ){1}
+              with:
+      Height: 
+        selector: //*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '身高')]
+        postProcess:
+          - replace:
+            - regex: ^(.*? ){1}
+              with:
+      # Measurements: //*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '胸圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '腰圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '臀圍')]//*[@id="waterfall"]/div[1]/div/div[2]/p[contains(text(), '罩杯')]
```
Comment (DogmaDragon) [inline scrapers/JavBus.yml:64]: Any reason this is commented out?

Comment (benny502) [reply scrapers/JavBus.yml:64]: I haven't found a good way to implement it yet.

**Comment by fengyichui** [comment]
This doesn't seem to work anymore on the latest Javbus website.

**Comment by Pheromir** [comment]
> This doesn't seem to work anymore on the latest Javbus website.

For me it worked adding
```yml
driver:
  headers:
    - Key: Accept-Language
      Value: de,en-US;q=0.7,en;q=0.3
```
to the yml. Without the "de" it didn't work and I only got a login window. Idk if they're checking if the IP address matches the value, I think you could open javbus in your browser, copy the header your browser is sending and paste it into the yml.

**Pull request merged**